RecoVertex/PrimaryVertexProducer/python/OfflinePrimaryVertices_cfi.py

```diff
@@ -48,6 +48,25 @@
                                         
 )
 
+from Configuration.ProcessModifiers.weightedVertexing_cff import weightedVertexing
+weightedVertexing.toModify(offlinePrimaryVertices,
+                           vertexCollections = cms.VPSet(
+                           [cms.PSet(label=cms.string(""),
+                                     algorithm=cms.string("WeightedMeanFitter"),
+                                     chi2cutoff = cms.double(2.5),
+                                     minNdof=cms.double(0.0),
+                                     useBeamConstraint = cms.bool(False),
+                                     maxDistanceToBeam = cms.double(1.0)
+                           ),
+                           cms.PSet(label=cms.string("WithBS"),
+                                     algorithm = cms.string('WeightedMeanFitter'),
+                                     minNdof=cms.double(0.0),
+                                     chi2cutoff = cms.double(2.5),
+                                     useBeamConstraint = cms.bool(False),
```

Comment:
is this supposed to be `False`? it looks like running the same algorithm twice

Reply:
Indeed, need to fix it